runtime/README.md

```diff
@@ -334,38 +356,13 @@ Caliptra Runtime firmware is responsible for initializing DPE’s Default Contex
 * Set flag in the TCI Node that this node was created by the DPE implementation.
   This will be used to set the VENDOR\_INFO field in TcbInfo to “VNDR”.
 
-*Note: the Runtime CDI will be read as-needed and will not be accessed during
-initialization.*
+*Note: the Runtime CDI (from KeyVault) will be used as-needed and will not be
+accessed during initialization.*
 
 ### TCI Storage
 
-Caliptra SHALL set `MAX_TCI_NODES` to 24. To support this, Caliptra will
-allocate 24 hardware PCRs to be exclusively used by DPE.
-
-These PCRs will store only the cumulative journey of the PCRs.
-
-#### PCR Properties
-
-* 48 bytes
-* Initial value of all zeros
-* Extend operation: `NEW_VALUE = SHA384_HASH(OLD_VALUE || MEASUREMENT)`
-* Clear operation: Resettable to initial value by Runtime Firmware
-
-#### DPE PCR Usage
```

Comment:
There are some other references to PCR (eg. table below) that should be rephrase - can scrub the rest of spec for those?

Reply:
Removed (note there are still some PCR references which refer to PCRs that hold measurements of Caliptra firmware)

Reply:
Thanks